doc/source/tune.rst

```diff
@@ -0,0 +1,160 @@
+Ray.tune: Efficient distributed hyperparameter search with Ray
+==============================================================
+
+This document describes Ray.tune, a hyperparameter tuning tool for long-running tasks such as RL and deep learning training. It has the following features:
+
+-  Early stopping algorithms such as `Median Stopping Rule <https://research.google.com/pubs/pub46180.html>`__ and `HyperBand <https://arxiv.org/abs/1603.06560>`__.
+
+-  Integration with visualization tools such as `TensorBoard <https://www.tensorflow.org/get_started/summaries_and_tensorboard>`__, `rllab's VisKit <https://media.readthedocs.org/pdf/rllab/latest/rllab.pdf>`__, and a `parallel coordinates visualization <https://en.wikipedia.org/wiki/Parallel_coordinates>`__.
+
+-  Flexible trial variant generation, including grid search, random search, and conditional parameter distributions.
+
+-  Resource-aware scheduling, including support for concurrent runs of algorithms that may themselves be parallel and distributed.
+
+
+Getting Started
+--------------- 
+
```

Comment:
We should have something that is explicit that says: 
"Here are three ways you can use tune: YAML, Python, Script"

I think also the script stuff is dropped (compared to previous docs)?

Reply:
Also we should have a small section that describes how to use Tune with RLLib

Reply:
Yeah I'd like to de-emphasize the script use case. It (and RLlib) is mentioned briefly in a later section. Perhaps you can elaborate on Tune+RLlib in the RLlib docs?

Reply:
Sure.